isb_web/manage.py

```diff
@@ -216,7 +220,13 @@ async def auth(request: starlette.requests.Request):
     token = await oauth.orcid.authorize_access_token(request)
     request.session["user"] = dict(token)
     redirect_url = url_utilities.joined_url(str(request.url), config.Settings().auth_response_redirect_fragment)
-    return starlette.responses.RedirectResponse(url=redirect_url)
+    # redirect_url = "http://localhost:3000/#/dois"
```

Comment:
For testing purposes, I used the redirect_url as localhost.

Reply:
For testing that works, please remove this line before merging.